lib/src/openid.dart

```diff
@@ -16,9 +16,9 @@ class OpenId {
   final RegExp _validation_regexp =
       RegExp(r'^https://steamcommunity.com/openid/id/(7[0-9]{15,25})$');
 
-  String _host;
-  String _returnUrl;
-  Map<String, String> _data;
+  String? _host;
+  String? _returnUrl;
+  late Map<String, String> _data;
```

Comment:
You should be able to make these variables `late final` (remeber to remove the custom getters afterwards and change all the usages from _<var> to <var> ie: _host to host)
``` 
late final String host;
late final String returnUrl;
late final Map<String, String> data;
``